packages/android_alarm_manager/android/src/main/java/io/flutter/plugins/androidalarmmanager/AlarmService.java

```diff
@@ -152,15 +152,6 @@ public static void setCallbackDispatcher(Context context, long callbackHandle) {
     prefs.edit().putLong(CALLBACK_HANDLE_KEY, callbackHandle).apply();
   }
 
-  public static boolean setBackgroundFlutterView(FlutterNativeView view) {
```

Comment:
Notice that this isn't really setting a `View`, it's setting a `FlutterNativeView`, which is now essentially represented by `FlutterEngine`. Are you sure this method can be deleted? Where is this behavior handled now?

Reply:
I think this is an artifact from the original implementation of this plugin when we were running all callbacks on the main isolate, which meant we had to be able to handle the case when the application was moved from foreground to background. This plugin now instead starts a background isolate immediately when `AndroidAlarmManager.initialize` is invoked or when the `AlarmService` is started via an `Intent`, and this isolate is kept alive until the service shuts down.